components/Frame/SecondaryNav.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react'
+import { css } from 'glamor'
+import { compose } from 'react-apollo'
+import { colors, mediaQueries } from '@project-r/styleguide'
+
+import { matchPath } from '../../lib/routes'
+import withT from '../../lib/withT'
+import NavLink from './Popover/NavLink'
+import Link from '../Link/Path'
+import Sections from '../Frame/Popover/Sections'
+
+import {
+  HEADER_HEIGHT,
+  HEADER_HEIGHT_MOBILE,
+  SUBHEADER_HEIGHT,
+  SUBHEADER_HEIGHT_MOBILE,
+  ZINDEX_HEADER
+} from '../constants'
+
+export const SecondaryNav = ({
+  secondaryNav,
+  dark,
+  isFront,
+  showSecondary,
+  router,
+  hasOverviewNav,
+  t
+}) => {
+  const active = matchPath(router.asPath)
+  return (
+    <>
+      {isFront || hasOverviewNav ? (
```

Comment:
You should actually set `hasOverviewNav` on the front page:

```suggestion
      {hasOverviewNav ? (
```